biojava-structure/src/test/java/org/biojava/nbio/structure/TestAltLocs.java

```diff
@@ -604,6 +593,164 @@ public void test3U7Tmmcif() throws IOException, StructureException{
+						"_atom_site.pdbx_PDB_model_num \n" +
+						"ATOM   102 N N   . ARG A 1 13 ? 9.889  23.379 13.115 1.00 6.57  ? 102  ARG A N   1\n" +
+						"ATOM   103 C CA  . ARG A 1 13 ? 9.540  23.003 14.482 1.00 7.05  ? 102  ARG A CA  1\n" +
+						"ATOM   104 C C   . ARG A 1 13 ? 10.407 23.758 15.489 1.00 6.88  ? 102  ARG A C   1\n" +
+						"ATOM   105 O O   . ARG A 1 13 ? 9.915  24.196 16.532 1.00 7.69  ? 102  ARG A O   1\n" +
+						"ATOM   106 C CB  . ARG A 1 13 ? 9.706  21.494 14.688 1.00 9.07  ? 102  ARG A CB  1\n" +
+						"ATOM   107 C CG  A ARG A 1 13 ? 8.757  20.644 13.854 0.50 14.39 ? 102  ARG A CG  1\n" +
+						"ATOM   108 C CG  B ARG A 1 13 ? 8.693  20.645 13.938 0.50 13.58 ? 102  ARG A CG  1\n" +
+						"ATOM   109 C CD  A ARG A 1 13 ? 9.109  19.164 13.950 0.50 18.14 ? 102  ARG A CD  1\n" +
+						"ATOM   110 C CD  B ARG A 1 13 ? 8.710  19.216 14.456 0.50 16.34 ? 102  ARG A CD  1\n" +
+						"ATOM   111 N NE  A ARG A 1 13 ? 8.983  18.644 15.310 0.50 20.72 ? 102  ARG A NE  1\n" +
+						"ATOM   112 N NE  B ARG A 1 13 ? 8.315  19.158 15.861 0.50 23.99 ? 102  ARG A NE  1\n" +
+						"ATOM   113 C CZ  A ARG A 1 13 ? 7.826  18.445 15.933 0.50 23.45 ? 102  ARG A CZ  1\n" +
+						"ATOM   114 C CZ  B ARG A 1 13 ? 8.404  18.072 16.622 0.50 24.56 ? 102  ARG A CZ  1\n" +
+						"ATOM   115 N NH1 A ARG A 1 13 ? 6.683  18.718 15.321 0.50 23.60 ? 102  ARG A NH1 1\n" +
+						"ATOM   116 N NH1 B ARG A 1 13 ? 8.881  16.942 16.118 0.50 28.42 ? 102  ARG A NH1 1\n" +
+						"ATOM   117 N NH2 A ARG A 1 13 ? 7.812  17.972 17.172 0.50 24.80 ? 102  ARG A NH2 1\n" +
+						"ATOM   118 N NH2 B ARG A 1 13 ? 8.013  18.115 17.888 0.50 26.52 ? 102  ARG A NH2 1\n";
+
+		SimpleMMcifParser parser = new SimpleMMcifParser();
+		SimpleMMcifConsumer consumer = new SimpleMMcifConsumer();
+		parser.addMMcifConsumer(consumer);
+
+		BufferedReader buf = new BufferedReader(new StringReader(mmcifData));
+		parser.parse(buf);
+		buf.close();
+
+		Structure s = consumer.getStructure();
+		Chain c = s.getPolyChains().get(0);
+		assertEquals(1, c.getAtomGroups().size());
+		Group g = c.getAtomGroup(0);
+		assertEquals(11, g.size());
+
+		// there's the main group (. and A) plus the 2 alt loc groups (A and B). The alt locs will contain all the '.' atoms too
+		assertEquals(2, g.getAltLocs().size());
+
+		for (Atom a : g.getAtoms()) {
+			if (a.getName().equals("C") || a.getName().equals("N") || a.getName().equals("O") || a.getName().equals("CA") || a.getName().equals("CB"))
+				assertEquals(' ', a.getAltLoc().charValue());
+			else
+				assertEquals('A', a.getAltLoc().charValue());
+		}
+
+		assertEquals(11, g.getAltLocs().get(0).size());
+		for (Atom a : g.getAltLocs().get(0).getAtoms()) {
+			if (a.getName().equals("C") || a.getName().equals("N") || a.getName().equals("O") || a.getName().equals("CA") || a.getName().equals("CB"))
+				assertEquals(' ', a.getAltLoc().charValue());
+			else
+				assertEquals('A', a.getAltLoc().charValue());
+		}
```

Comment:
So does the first altloc contain the same atoms as the main group? Even stronger, is `this == this.getAltLocs().get(0)`?